pokedex_GEN.go

```diff
@@ -0,0 +1,1010 @@
+	{ NatDex: 337 },
+	{ NatDex: 338 },
+	{ NatDex: 339 },
+	{ NatDex: 340 },
+	{ NatDex: 341 },
+	{ NatDex: 342 },
+	{ NatDex: 343 },
+	{ NatDex: 344 },
+	{ NatDex: 345 },
+	{ NatDex: 346 },
+	{ NatDex: 347 },
+	{ NatDex: 348 },
+	{ NatDex: 349 },
+	{ NatDex: 350 },
+	{ NatDex: 351 },
+	{ NatDex: 352 },
+	{ NatDex: 353 },
+	{ NatDex: 354 },
+	{ NatDex: 355 },
+	{ NatDex: 356 },
+	{ NatDex: 357 },
+	{ NatDex: 358 },
+	{ NatDex: 359 },
+	{ NatDex: 360 },
+	{ NatDex: 361 },
+	{ NatDex: 362 },
+	{ NatDex: 363 },
+	{ NatDex: 364 },
+	{ NatDex: 365 },
+	{ NatDex: 366 },
+	{ NatDex: 367 },
+	{ NatDex: 368 },
+	{ NatDex: 369 },
+	{ NatDex: 370 },
+	{ NatDex: 371 },
+	{ NatDex: 372 },
+	{ NatDex: 373 },
+	{ NatDex: 374 },
+	{ NatDex: 375 },
+	{ NatDex: 376 },
+	{ NatDex: 377 },
+	{ NatDex: 378 },
+	{ NatDex: 379 },
+	{ NatDex: 380 },
+	{ NatDex: 381 },
+	{ NatDex: 382 },
+	{ NatDex: 383 },
+	{ NatDex: 384 },
+	{ NatDex: 385 },
+	{ NatDex: 386 },
+	{ NatDex: 387 },
+	{ NatDex: 388 },
+	{ NatDex: 389 },
+	{ NatDex: 390 },
+	{ NatDex: 391 },
+	{ NatDex: 392 },
+	{ NatDex: 393 },
+	{ NatDex: 394 },
+	{ NatDex: 395 },
+	{ NatDex: 396 },
+	{ NatDex: 397 },
+	{ NatDex: 398 },
+	{ NatDex: 399 },
+	{ NatDex: 400 },
+	{ NatDex: 401 },
+	{ NatDex: 402 },
+	{ NatDex: 403 },
+	{ NatDex: 404 },
+	{ NatDex: 405 },
+	{ NatDex: 406 },
+	{ NatDex: 407 },
+	{ NatDex: 408 },
+	{ NatDex: 409 },
+	{ NatDex: 410 },
+	{ NatDex: 411 },
+	{ NatDex: 412 },
+	{ NatDex: 413 },
+	{ NatDex: 414 },
+	{ NatDex: 415 },
+	{ NatDex: 416 },
+	{ NatDex: 417 },
+	{ NatDex: 418 },
+	{ NatDex: 419 },
+	{ NatDex: 420 },
+	{ NatDex: 421 },
+	{ NatDex: 422 },
+	{ NatDex: 423 },
+	{ NatDex: 424 },
+	{ NatDex: 425 },
+	{ NatDex: 426 },
+	{ NatDex: 427 },
+	{ NatDex: 428 },
+	{ NatDex: 429 },
+	{ NatDex: 430 },
+	{ NatDex: 431 },
+	{ NatDex: 432 },
+	{ NatDex: 433 },
+	{ NatDex: 434 },
+	{ NatDex: 435 },
+	{ NatDex: 436 },
+	{ NatDex: 437 },
+	{ NatDex: 438 },
+	{ NatDex: 439 },
+	{ NatDex: 440 },
+	{ NatDex: 441 },
+	{ NatDex: 442 },
+	{ NatDex: 443 },
+	{ NatDex: 444 },
+	{ NatDex: 445 },
+	{ NatDex: 446 },
+	{ NatDex: 447 },
+	{ NatDex: 448 },
+	{ NatDex: 449 },
+	{ NatDex: 450 },
+	{ NatDex: 451 },
+	{ NatDex: 452 },
+	{ NatDex: 453 },
+	{ NatDex: 454 },
+	{ NatDex: 455 },
+	{ NatDex: 456 },
+	{ NatDex: 457 },
+	{ NatDex: 458 },
+	{ NatDex: 459 },
+	{ NatDex: 460 },
+	{ NatDex: 461 },
+	{ NatDex: 462 },
+	{ NatDex: 463 },
+	{ NatDex: 464 },
+	{ NatDex: 465 },
+	{ NatDex: 466 },
+	{ NatDex: 467 },
+	{ NatDex: 468 },
+	{ NatDex: 469 },
+	{ NatDex: 470 },
+	{ NatDex: 471 },
+	{ NatDex: 472 },
+	{ NatDex: 473 },
+	{ NatDex: 474 },
+	{ NatDex: 475 },
+	{ NatDex: 476 },
+	{ NatDex: 477 },
+	{ NatDex: 478 },
+	{ NatDex: 479 },
+	{ NatDex: 480 },
+	{ NatDex: 481 },
+	{ NatDex: 482 },
+	{ NatDex: 483 },
+	{ NatDex: 484 },
+	{ NatDex: 485 },
+	{ NatDex: 486 },
+	{ NatDex: 487 },
+	{ NatDex: 488 },
+	{ NatDex: 489 },
+	{ NatDex: 490 },
+	{ NatDex: 491 },
+	{ NatDex: 492 },
+	{ NatDex: 493 },
+	{ NatDex: 0 },
```

Comment:
do you know what caused all of these `0` entries to be created?

Reply:
Other forms of pokemon like deoxys and rotom are listed as separate pokemon

Reply:
oh, right. do you foresee this causing any issues with our implementation of getting pokemon names? (the one in `pokemon.go`)

Reply:
Not really. We can worry about pokemon forms later.